data/ibukunoluwanap.json

```diff
@@ -0,0 +1,50 @@
+{
+   "name":"Ibukunoluwa Naphtali",
+   "displayStatsPublic":true,
+   "type":"personal",
+   "bio":"Mobile Developer | Tech Educator | Open Source | Work with Flutter, ReactJS And NodeJS.",
+   "username":"ibukunoluwanap",
```

Comment:
Not 100% sure this field is valid/needed?

Reply:
Not needed Ig, it would be better to remove this one. 🛠

Hey @ibukunoluwanap, please remove this username field from line no. `6`.

Reply:
Ok